tools/local-network-setup/.bootstrap_origintrail_noderc

```diff
@@ -0,0 +1,25 @@
+{
+  "blockchain":[
+    {
+      "blockchainTitle": "Polygon",
+      "networkId": "polygon::testnet",
+      "rpcEndpoints": ["https://rpc-mumbai.maticvigil.com/"],
+      "publicKey": "0xBD084aB97C704FE4a6D620CB7c30C0BE0366646f",
+      "privateKey": "0x61b1b921f6d57d7f60c4ac17d5bd51f8394c5d72b8d4db177b5d419fbc5c7a41"
```

Comment:
We should not commit private keys that are used for testnet environment - this should be read from .env file

Reply:
This are keys for starting local environment, they are not used in testnet environment

Reply:
Yes but we are using real testnet for starting local environment right? I think that we should keep private keys only if we are using ganache or something like that, but we should ever commit real keys that are used for real testnet. @kotlarmilos what do you think?

Reply:
I didn't understood you first time. Yes, I will remove this.